recipes/pysdl2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "PySDL2" %}
+{% set version = "0.9.4" %}
+{% set md5 = "720b33ed048c169643406ae39ca4dd8d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  patches:
+    - locate_sdl.patch
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
This breaks the builds on all platforms, because it doesn't know the `--single-version-externally-managed ` flag:
~~~
C:\Miniconda35-x64\conda-bld\work\PySDL2-0.9.4>call activate.bat C:\Miniconda35-x64\envs\_build 
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help
error: option --single-version-externally-managed not recognized
Command failed: cmd.exe /c bld.bat
Command exited with code 1
~~~
(example from appveyor, but Travis and Circle crash on the same)